client/src/components/Pages/Structure/Builder/Builder.tsx

```diff
@@ -183,162 +244,193 @@ const Builder: React.FC = () => {
     }
   };
 
-  /**
-   * Render nomenclature preview footer underneath builder interface
-   * @returns completed footer element
-   */
-  const renderFooterBar = () => (
-    <div className="hr-section">
-      {fusion.structural_elements
-        ?.filter(
-          (element: ClientElementUnion) =>
-            Boolean(element) && element.nomenclature
-        )
-        .map((element: ClientElementUnion, index: number) => (
-          <div key={element.element_id}>{`${index ? "::" : ""}${
-            element.nomenclature
-          }`}</div>
-        ))}
-    </div>
+  const nomenclatureContent = fusion.structural_elements
+    .filter(
+      (element: ClientElementUnion) => Boolean(element) && element.nomenclature
+    )
+    .map(
+      (element: ClientElementUnion, index: number) =>
+        `${index ? "::" : ""}${element.nomenclature}`
+    );
```

Comment:
The `::` divider can actually change to `(::)` based on the value of `fusion.evidence`: https://fusions.cancervariants.org/en/latest/nomenclature.html#inferred-fusions

That's a little weird because it's based on a property that isn't set on this page, but might also be confusing to see it differently here? Idk.

Reply:
Oh, I'll add that I almost think - for reasons like this - you might not want to include the divider nomenclature at all on this page. Alex probably feels strongly otherwise (and I ofc would defer in that case), but I think it could be potentially misleading to provide this sort of nomenclature preview up front, when there are contributing elements on other pages (this and the regulatory element tab). I used MUI Chips on the summary tab to show elements without putting them in the nomenclature, maybe that's an option here?